src/client/js/recruitmentSurvey.js

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const recruitmentBanner = document.querySelector('.recruitment-banner')
+const recruited = document.cookie.split('; ').some((item) => item.trim().startsWith('recruited='))
+
+function init () {
+  if (!recruitmentBanner) {
+    return
+  }
+
+  // Only show the banner and set event listeners if the user has not interacted with the survey
+  if (!recruited) {
+    recruitmentBanner.toggleAttribute('hidden')
+    const recruitmentBannerLink = document.getElementById('recruitment-banner')
+    const recruitmentDismissButton = document.getElementById('recruitment-dismiss')
+    recruitmentBannerLink.addEventListener('click', handleEvent)
+    recruitmentDismissButton.addEventListener('click', handleEvent)
+  }
+}
+
+function handleEvent (e) {
+  const date = new Date()
+  date.setTime(date.getTime() + 30 * 24 * 60 * 60 * 1000)
+  document.cookie = 'recruited=true; expires=' + date.toUTCString() + '; path=/'
+
+  switch (true) {
+    case e.target.matches('#recruitment-dismiss'):
+      document.getElementById('recruitment-banner').parentElement.remove()
+      break
```

Comment:
ux note: We only remove the link when the user dismisses the banner (until the page refreshes)